pulpcore/cli/common/generic.py

```diff
@@ -132,36 +160,36 @@ def load_json_callback(
 ##############################################################################
 # Decorator common options
 
+def pulp_option(*args: Any, **kwargs: Any) -> Callable[..., Any]:
```

Comment:
I tried for 2 hours to make mypy happy here but no luck. The type from click is a bit complex:

https://github.com/pallets/click/blob/ffeddbaf0c8226201136593d29324c951f2b2fa3/src/click/decorators.py#L229

Reply:
I think TypeVar can help to describea callable that returnes the same type as it received.

Reply:
```
_F = TypeVar("_F")
def pulp_option(*args: Any, **kwargs: Any) -> Callable[[_F], _F]:
```
works for me.

Reply:
Huh, could have sworn I tried that at one point. It works. Thank you.